internal/pkg/plugin/resolver_test.go

```diff
@@ -2,10 +2,10 @@ package plugin
 
 import (
 	"fmt"
+	"github.com/cyberark/secretless-broker/pkg/secretless/config/v1"
```

Comment:
Ditto moving internal import to bottom of import list.
Ditto on naming this import config_v1